yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -553,6 +552,35 @@ func (p *ParserIssueDetector) FinalizeColumnMetadata() {
 
 	// Build partition hierarchies
 	p.buildPartitionHierarchies()
+
+	p.buildUsageCategoryForAllTables()
+}
+
+func (p *ParserIssueDetector) buildUsageCategoryForAllTables() {
```

Comment:
what about indexes? do we not lookup usages for indexes directly for any of the issues?

Reply:
there is a function for indexes also in parser Issue detector `getUsageCategoryForIndex`